package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "react-webpack-from-scratch",
+  "version": "1.0.0",
+  "description": "React app made from scratch without CRA",
+  "main": "./src/index.tsx",
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open --hot",
+    "build": "webpack --mode production",
+    "build:dev": "webpack --mode production --env development",
```

Думаю, что тут для **mode** нужен `development`:
```suggestion
    "build:dev": "webpack --mode development --env development",
```

---

Немножко не так, webpack --mode development - это старт дев сервера, в задании же указано именно чтобы билд проводился под разные энвароменты, типа один билд для продакш сервера (доступного паблик), другой например чтобы фича бранчи хранить (с текущей версией в разработке)